data/validation/selectioncount.go

```diff
@@ -0,0 +1,20 @@
+package validation
+
+import (
+	"fmt"
+
+	"fyne.io/fyne/v2"
+)
+
+// NewSelectionCount returns a fyne.SelectionValidator that requires
+// that "required" amount or more items are selected.
+func NewSelectionCount(required int) fyne.SelectionValidator {
```

Comment:
I just realised this assumes that validation should only check a minimum. I would think that a maximum is more common, should this take 2 parameters instead?

Reply:
Seems like a good idea. I'll get it updated.